### DISCUSSION TITLE
StickPM - Social interactions with chatters and streamers alike, a place to create your own room and grow a space you feel at home within while in a social environment. Interact with users and streamers and engage in mini games with spectators.

### DISCUSSION THREAD

**Discussion by BadNintendo** [community/community#142072] [Other Feature Feedback, Questions, & Ideas]
![Screenshot 2024-10-19 112213](https://github.com/user-attachments/assets/635b850e-8681-4709-bd8c-d043b5a806d4)
![Screenshot 2024-10-20 123240](https://github.com/user-attachments/assets/b3285a26-0434-4e68-a826-93c278288483)
### Select Topic Area

Show & Tell

### Join the StickPM Project: Streamline Your Web Development Skills!

Greetings, fellow developers and designers!

I'm excited to introduce **StickPM**, a cutting-edge project aimed at creating a robust streaming chatroom platform. Inspired by the features of BlogTV, Twitch, and Stickam, StickPM is on its way to revolutionizing how we interact online. As the sole developer, I recognize that bringing this vision to life requires collaboration with passionate individuals who share a dream for a vibrant online community.

#### About StickPM

StickPM is designed to provide a seamless and interactive streaming experience. The backend is nearly complete, and I am actively developing the client-side to ensure a smooth demo version for users. Our platform will enable users to broadcast, chat, and manage their streams effortlessly.

#### Key Features:

- **Interactive Streaming**: Real-time video streaming with minimal latency.
- **Chatroom Functionality**: Dynamic chatrooms with real-time messaging.
- **User Management**: Robust user and room management systems.
- **Security**: Advanced security features, including HTTPS enforcement and user authentication.
- **Polling and Reactions**: Interactive polls and reactions for engaging user experience.
- **Stream Management**: Easy-to-use tools for starting, stopping, and managing streams.
- **Push-to-Talk**: Convenient push-to-talk feature for audio management during streams.

#### Why Join Us?

- **Collaborative Environment**: Work alongside me and other talented developers and designers to shape StickPM.
- **Innovative Project**: Be part of a project that aims to redefine streaming experiences.
- **Skill Enhancement**: Sharpen your skills in web development, security, and real-time communication.
- **Open Source Contribution**: Make meaningful contributions to an open-source project.

#### Who Are We Looking For?

- **Web Developers**: Experienced with Node.js, Express, EJS, and Socket.io.
- **UI/UX Designers**: Talented in creating intuitive and engaging user interfaces.
- **Backend Developers**: Skilled in server-side logic, database management, and security.
- **Discord Developers**: Interested in enhancing our Discord server, currently called Chill Spot, which will soon be rebranded to align with StickPM.
- **General Enthusiasts**: Passionate about streaming technology and web development.

#### Join Our Community!

We have a dedicated Discord server for all project discussions, collaborations, and updates. I welcome all developers, designers, and anyone interested in contributing to join us on Discord.

[Join StickPM on Discord](https://discord.gg/fxVbabpf)

#### Next Steps:

1. **Join the Discord Server**: Connect with me and the team to get started on contributing to StickPM.
2. **Familiarize Yourself**: Review the existing codebase and understand the project structure.
3. **Start Contributing**: Dive into coding, designing, or enhancing our Discord server.

I look forward to welcoming you to the StickPM team. Let's create something amazing together!

#### Contact Information:

For any questions or additional information, feel free to reach out on Discord. Join us, and be part of the future of streaming technology!

Thank you for considering joining StickPM. Together, we can achieve incredible things!

Warm regards,

The StickPM Team

**Comment by LiteBrite82** [comment]
Hi @BadNintendo , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
